src/lib/drivers/driver.js

```diff
@@ -86,6 +90,31 @@ class DriverBase {
     return Promise.reject(new Error('Not implemented'));
   }
 
+  evaluateAsync(asyncExpression) {
+    return new Promise((resolve, reject) => {
+      let asyncTimeout;
+
+      // Inject the call to capture inspection.
+      const expression = `(function() {var __inspect = inspect;${asyncExpression}})()`;
```

Comment:
we run in Chrome 52+, so you should be good to use `const` in the page context as well